src/main/java/techreborn/blockentity/machine/tier1/GreenhouseControllerBlockEntity.java

```diff
@@ -81,7 +92,7 @@ private void workCycle() {
 			) {
 				if (getStored(EnergySide.UNKNOWN) > TechRebornConfig.greenhouseControllerEnergyPerBonemeal) {
 					useEnergy(TechRebornConfig.greenhouseControllerEnergyPerBonemeal);
-					blockState.scheduledTick((ServerWorld) world, blockPos, world.random);
+					blockState.randomTick((ServerWorld) world, blockPos, world.random);
```

Comment:
These are same
![image](https://user-images.githubusercontent.com/1382352/123606967-1c3b8c00-d806-11eb-9241-0418a11a6320.png)

Reply:
My understanding is that is the default behavior. I think CropBlock (and similar types) override that. Also in my testing there was only noticeable growth with `randomTick`
(From CropBlock.java)
![image](https://user-images.githubusercontent.com/27027656/123690371-4a2ec980-d822-11eb-80f4-8b319e7e5151.png)

Reply:
For instance this is for Cactus:
![image](https://user-images.githubusercontent.com/27027656/123691252-61ba8200-d823-11eb-88be-b5b58bbd56ea.png)

Reply:
Yeah, sorry, I haven't checked crop blocks

Reply:
np, thanks for accepting the PR!